### DISCUSSION TITLE
How to show the relationship of clicked node?

### DISCUSSION THREAD

**Discussion by yugeshda** [neo4j-contrib/neovis.js#362] [Q&A]
Trying to show the relationship of the specific node after clicking the node. Am using **Movie** sample data for learning purposes. Limit the query to **2**. Now got two nodes with movie name **The Matrix** and **The Matrix Reloaded**, want to show the relationship of the **The Matrix** node only, not for **The Matrix Reloaded.**. How it was visible in Neo4j browser after hover and select the graph.

**Expected Result After click the _The Matrix_ node in neovis.js**
![image](https://github.com/neo4j-contrib/neovis.js/assets/65076391/fbbd2bf1-2d99-4688-837a-1bd6f45dc1bb)

**Comment by thebestnom** [comment]
Do you mean double click to expend? I don't understand the problem 😅
If you mean expend you'll have to use click event and then do updateWithQuery
And if you mean only showing do it in the initial query 😅

**Reply by yugeshda** [reply]
@thebestnom Initially there will be two nodes without any relationship from this query `MATCH (n:Movie) WITH n limit 2`.  After clicked any one of the node to show the relationship of the clicked node. Am tried with the query `MATCH (n:Movie) WITH n limit 2 OPTIONAL MATCH (n:Movie{title:'The Matrix'})-[r]-(b) RETURN n,r,b` but it creates the relationship for both nodes.

![image](https://github.com/neo4j-contrib/neovis.js/assets/65076391/115d07a7-c6b7-4396-ae92-310a38947324)
